Contribute/index.md

```diff
@@ -0,0 +1,140 @@
+
+#### Creating a PR through a fork
+There is a lot of great guides on how to set up a fork and work with forks, so below there will be a quick guide on how to do it. If you wish for some more general information, Github has a [guide here](https://help.github.com/articles/fork-a-repo/).
+
+**Step 1: Creating a fork** 
+
+A fork of the Umbraco Documentation repository means that you get a copy of the whole repository on your own Github profile. You can create a fork by simply clicking a button:
+![Creating a fork](images/fork-repository.png)
+
+Once the fork has been created you will have your own copy of the documentation. If you clone your fork down you can make changes to a lot of things and then push all the changes up to your fork. 
+
+When you are done and happy with the changes you've made, you can submit a pull request where you try to sync your copy with the "real" repository:
+![Fork of documentation](images/example-of-fork.png)
+
+**Step 2: Syncing your fork**
+
+Sometimes - like in the example above - you may end up forking and then not working on the documentation for a while. Once you start you are some commits behind, so before making changes and making a pull request you should do a rebase. For this you set the main repository as an upstream to sync from, fetch the updates and update your own fork to ensure you are in sync:
+
+```cmd
+git remote add upstream https://github.com/umbraco/UmbracoDocs/
+git fetch upstream
+git rebase upstream/master
+```
+
+**Step 3: Creating a PR**
+
+If you do this locally and then push it to your fork you will have a synced up fork to start working with! Once you have made some changes and you are happy with the result, you can create a pull request (you may have to rebase again and resolve merged conflicts if a lot of things has been merged in since your last sync).
+![Creating a PR](images/pull-request.png)
+
+And that is all you need to do to create a fork, sync it and make a pull request to the main repository! 
+
+
+#### Labels for PRs
+- `Dependant On Release` - this label is for documentation pull requests that are made in advance of a new Umbraco release, will be merged in once the changes have been released
+- `need-OUR-update` - this label is used for PR's awaiting some new functionality being added to the Our Umbraco website
+- `D-Team` - this label is usually used when the PR is about something with Umbraco Core that the documentation team is not familiar with, meaning it requires a review from the Umbraco development team
+- `Umbraco Cloud`, `Umbraco Forms`, etc - if the issue is specific to something other than Umbraco CMS it will get the relevant Umbraco product label
+
+## Markdown conventions
+The Umbraco Documentation uses Markdown for all of the documentation - but more precisely we use the CommonMark specification, read more about the difference [here](https://commonmark.org/).
```

Comment:
Always try to avoid "here" links to improve accessibility of visually impaired people with screenreaders. They can ask for all links on a page (or a section).

Better: 

Read more about [the difference between markdown and commonmark](https://commonmark.org).